e2e/rbd.go

```diff
@@ -378,6 +378,27 @@ var _ = Describe("RBD", func() {
 					if err != nil {
 						e2elog.Failf("failed to create PVC with error %v", err)
 					}
+					app, err := loadApp(appPath)
+					if err != nil {
+						Fail(err.Error())
+					}
+					// write data in PVC
+					app.Namespace = f.UniqueName
+					app.Spec.Volumes[0].PersistentVolumeClaim.ClaimName = pvc.Name
+					wErr := writeDataInPod(app, f)
+					if wErr != nil {
+						Fail(wErr.Error())
+					}
+					opt := metav1.ListOptions{
+						LabelSelector: "app=write-data-in-pod",
+					}
+					e2elog.Logf("calculating checksum value")
+					checkSum, err := calculateSHA512sum(f, app, &opt)
+					e2elog.Logf("checksum value is %s", checkSum)
+					if err != nil {
+						e2elog.Failf("failed to calculate checksum with error %v", err)
```

Comment:
the app need to be deleted before deleting the pvc, looks  like we are not deleting the application